pyscriptjs/src/components/base.ts

```diff
@@ -263,16 +254,14 @@ function createWidget(name: string, code: string, klass: string) {
         }
 
         registerWidget() {
-            const pyodide = runtime;
             console.log('new widget registered:', this.name);
-            pyodide.globals.set(this.id, this.proxy);
+            runtime.globals.set(this.id, this.proxy);
         }
 
         async eval(source: string): Promise<void> {
-            const pyodide = runtime;
             try {
-                const output = await pyodide.runPythonAsync(source);
-                this.proxyClass = pyodide.globals.get(this.klass);
+                const output = await runtime.run(source);
+                this.proxyClass = runtime.globals.get(this.klass);
```

Comment:
This is going to be a problem in the future when we add different runtimes. Nothing to do in this PR but just for us to be aware.

Reply:
noted.